src/queryEngine.ts

```diff
@@ -130,10 +132,13 @@ export async function queryEngine(): Promise<void> {
     let remainingPlugins: String[] = []
     // loop over every app
     for (const app of apps) {
+      if (config.soloAppId != null && config.soloAppId !== app.appId) continue
```

Comment:
These new config props should be documented in `config.sample.json` (It'd be nice to convert to using cleaner-config).